packages/swipe-carousel/tsconfig.json

```diff
@@ -16,10 +16,9 @@
     "noEmit": true,
     "jsx": "react-jsx",
     "paths": {
-      "@utils": ["../../utils/index.ts"]
+      "@utils": ["../utils/index.ts"]
     }
   },
   "include": ["src"],
   "exclude": ["dist"],
-  "references": [{ "path": "./tsconfig.node.json" }]
```

Comment:
this was generated automatically by `vite` -- i wasn't exactly sure why it was included, but why is it deleted here?

Reply:
same ^